pkg/bootstrap/docker/up.go

```diff
@@ -1033,8 +1033,13 @@ func (c *ClientStartConfig) InstallServiceCatalog(out io.Writer) error {
 	if len(publicMaster) == 0 {
 		publicMaster = c.ServerIP
 	}
+	tag := c.ImageVersion
+	if len(c.ImageVersion) == 0 {
```

Comment:
I don't think this condition would ever be true, given that if we don't get one in the command line we default to the one compiled with the oc binary. However, if it was ever true, you would have to prepend "v" to the version string you get below